internal/core/app_deploy.go

```diff
@@ -135,32 +135,34 @@ type deployOperation struct {
 	// Set by init
 	autoHostname pb.UpsertDeploymentRequest_Tristate
 
-	// component is initialized and recycled at various points in the deployment
+	// Component is initialized and recycled at various points in the deployment
 	// operation as we have more templating information for the config.
-	component *Component
+	Component *Component
```

Comment:
Is there a reason behind this change related to adding a deployment URL?

If there is not a reason, OR its not related to deployment URLs, let's table this for another issue to discuss. This feels unrelated to this PR.

Reply:
Unrelated, sorry. It was just a change to be consistent with:
https://github.com/swisscom/waypoint/blob/feature/deployment-url/internal/core/app_release.go#L152